aiohttp/websocket_client.py

```diff
@@ -87,7 +87,9 @@ def ws_connect(url, protocols=(), timeout=10.0, connector=None,
     reader = resp.connection.reader.set_parser(WebSocketParser)
     writer = WebSocketWriter(resp.connection.writer, use_mask=True)
 
-    return ClientWebSocketResponse(
+    response_class = response_class or ClientWebSocketResponse
```

Comment:
Please use 

    if response_class is None:
        response_class = ClientWebSocketResponse idiom